src/app/features/registries/components/custom-step/custom-step.component.html

```diff
@@ -47,10 +51,10 @@ <h3 class="mb-2">
                 </h3>
               </div>
               @if (q.paragraphText) {
-                <p class="mb-3">{{ q.paragraphText }}</p>
+                <p class="mb-3 preserve-whitespace">{{ q.paragraphText }}</p>
               }
-              @if (q.helpText && q.fieldType !== FieldType.Radio && q.fieldType !== FieldType.Checkbox) {
-                <p>{{ q.helpText }}</p>
+              @if (q.helpText) {
```

Comment:
Does this update the issue outlined in the ticket ("Another one that’s not showing up is the `help_text`..." ). I think the root cause of the problem that's outlined in that ticket is not addressed where we use a `currentQuestion` object to store `helpText`, but this `helpText` can be overwritten in subsequent blocks.